cmd/dcrdata/views/treasury.tmpl

```diff
@@ -67,13 +71,6 @@
         <div class="position-relative d-flex justify-content-between align-items-start flex-wrap">
           <span>See also the <a href="/address/{{$.DevAddress}}">legacy treasury</a> during the transition to this decentralized treasury account.</span>
         </div>
-        <div class="row pb-2">
-          {{- if ne $bal.ImmatureCount 0}}
-          <div class="col-12 py-2 fs14 text-secondary text-left" data-target="address.numUnconfirmed">
-            <span class="font-weight-bold">Immature</span>: <span class="addr-unconfirmed-count">{{$bal.ImmatureCount}}</span>
-          </div>
-          {{- end}}
-        </div>
```

Comment:
This is almost always just going to say "immature: 256" now, unless there are immature tadds too. Can bring it back if desired.